geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -297,7 +297,7 @@ public StoppableCountDownLatch getInitializationLatchAfterGetInitialImage() {
   private Object regionUserAttribute;
 
   // TODO: shouldn't this be an identity map whose key is a RegionEntry?
```

Comment:
Can we address and remove this TODO?

Reply:
I removed the TODO. The use of the map doesn't really match well and we were using a ConcurrentHashMap. I don't think I would want to change the impl unless there's an open bug but I can't find any.